web/public/js/services/intercom-service.js

```diff
@@ -62,12 +69,8 @@ angular.module('cpZenPlatform').factory('intercomService', function ($localStora
       if (user) {
         if (userIsCDFAdmin(user)) {
```

Comment:
@tangentfairy from what I can see I think you can remove this `if (userIsCDFAdmin(user)) {` and leave only this 
`if(!userIsNinja(user)){`. Otherwise looks good :)